src/setupWorker/start/createStart.ts

```diff
@@ -61,6 +64,22 @@ export const createStart = (context: SetupWorkerInternalContext) => {
       const [worker, registration] = instance
 
       if (!worker) {
+        if (options?.findWorker) {
+          console.error(`\
+[MSW] Failed to locate the Service Worker registration using a custom "findWorker" predicate.
+
+Please ensure that the custom predicate properly locates the Service Worker registration at ("${resolvedOptions.serviceWorker.url}").
```

Comment:
Sorry for a nitpick, may I please ask to remove these parenthesis? It's a leftover from my side, should just say `registration at "${resolvedOptions.serviceWorker.url}"`. Otherwise it breaks the sentence.